src/config/constants.ts

```diff
@@ -187,7 +189,7 @@ export const AppConfigStatus = {
 
 export const FullRoutes = {
     LOGIN: `${Routes.LOGIN}`,
-    CENTRAL: 'https://api.devtron.ai',
+    CENTRAL: 'https://api-stage.devtron.ai',
```

Comment:
Change API to prod one

Reply:
@vivek-devtron As discussed, change this API back to the prod one after testing & before merging.